lib/qb_backend/posts/bookmark.ex

```diff
@@ -0,0 +1,60 @@
+defmodule QbBackend.Posts.Bookmark do
+  @moduledoc """
+  This module defines the schema for the bookmark section
+  """
+  use QbBackend.Utils.Schema
+
+  alias QbBackend.{
+    Posts.Manual,
+    Accounts.Profile,
+    Posts.Bookmark
+  }
+
+  schema "bookmarks" do
```

Comment:
@WanjikuMac  could you perhaps consider giving a bookmark a name so maybe a user can have many bookmarks and the ability to choose from one

Reply:
By bookmark name do you mean e.g `bookmark 1`? Otherwise the bookmark name is the name of the manual. Like when you book mark a post  you retrieve it by the name of the manual

Reply:
@WanjikuMac  hmmmm so if I as a profile own a bookmark I will expect it to have a name like I could have a `Yoga` bookmark and I could put yoga manuals I am interested in that bookmark but later down the line I may want a `business` bookmark so I can put the business articles I am intrested in it there ... do you see

Reply:
@WanjikuMac  no I mean the bookmark should have a field for the `name` of the bookmark, Consider the case where one person wants many bookmarks for example. I could have a bookmark that stores all the `Yoga` stuff I want to read and I could name it `my yoga reads` and I could have a bookmark that stores all the `Business` stuff I want to read and this one I could call `kazi` does this make sense?

Reply:
@zacck Aaah, I see.

Reply:
@WanjikuMac  no I mean the bookmark should have a field for the `name` of the bookmark, Consider the case where one person wants many bookmarks for example. I could have a bookmark that stores all the `Yoga` stuff I want to read and I could name it `my yoga reads` and I could have a bookmark that stores all the `Business` stuff I want to read and this one I could call `kazi` does this make sense?